### DISCUSSION TITLE
New logo for myMPD

### DISCUSSION THREAD

**Discussion by jcorporation** [jcorporation/myMPD#377] [General]
I am playing around with Inkscape to create a new myMPD logo. Here is the first draft. Any comments?

![myMPD-logo-draft1](https://user-images.githubusercontent.com/765844/100554059-8dc45380-3292-11eb-8562-abc99dc4b98f.png)

**Comment by john-** [comment]
Looks good.   Simple yet distinct.   Flows nicely too.
